python/heterocl/compute_api.py

```diff
@@ -921,5 +921,57 @@ def reduce_body():
     make_reduce.__doc__ = doc_str.format(name)
     return make_reduce
 
+
+def bitcast(tensor, dst_dtype, name=None):
+    """Bitcast a HeteroCL tensor to the destination data type of the same bitwidth.
+
+    This API **bitcast** the input tensor from its own data type (source dtype)
+    to the destination data type (dst_dtype). The destination data type must have
+    the same bitwidth with the source datatype. 
+
+    Parameters
+    ----------
+    tensor: Tensor or Expr
+        The input tensor or expression of the source data type
+    dst_dtype: Type
+        The destination data type. For example, hcl.UInt(32)
+    Name: str
+        The name of the returned tensor
```

Comment:
Add an empty line between the arguments.

Reply:
Done